buildpack.toml

```diff
@@ -27,7 +27,7 @@ api = "0.7"
     sha256 = "d59b0711914a7876a10032d925811b9cfe87a1e9f87c82a861c3d69bb6857e93"
     source = "https://rubygems.org/downloads/bundler-1.17.3.gem"
     source_sha256 = "bc4bf75b548b27451aa9f443b18c46a739dd22ad79f7a5f90b485376a67dc352"
-    stacks = ["io.buildpacks.stacks.bionic"]
```

Comment:
I'm slightly confused why only one dependency is marked with Jammy support. Also I see we're adding the Jammy stack support in this PR. Will there be a separate PR to regenerate all the other dependencies (so they're marked with Jammy support) and to add testing?

Reply:
We can just leave this as-is if you're planning to essentially nuke the buildpack.toml so that all of the versions are regenerated with Jammy support

Reply:
I'd vote for rebuilding all dependencies with the new logic and jammy support. So I'd be happy to merge this as-is with the understanding we would rebuild the dependencies before shipping.